src/partials/header-content.hbs

```diff
@@ -19,34 +19,63 @@
       </div>
       <div class="navbar-end">
         <div class="navbar-item has-dropdown is-hoverable">
-          <div class="navbar-link">Processors</div>
+          <div class="navbar-link">AsciiDoc</div>
           <div class="navbar-dropdown">
-            <div class="navbar-item"><strong>Asciidoctor</strong></div>
-            <a class="navbar-item" href="https://github.com/asciidoctor/asciidoctor" target="_blank" rel="noopener">Source</a>
-            <a class="navbar-item" href="https://github.com/asciidoctor/asciidoctor/issues" target="_blank" rel="noopener">Issues</a>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoc::index.adoc')}}">Language</a>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoc::syntax-quick-reference.adoc')}}">Syntax Quick Reference</a>
+          </div>
+        </div>
+        <div class="navbar-item has-dropdown is-hoverable">
+          <div class="navbar-link">Processing</div>
+          <div class="navbar-dropdown">
+            <div class="navbar-item"><strong>Core Processors</strong></div>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoctor::index.adoc')}}">Asciidoctor <small>Ruby</small></a>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoctor.js::index.adoc')}}">Asciidoctor.js <small>JavaScript</small></a>
+            <a class="navbar-item" href="#">AsciidoctorJ <small>Java</small></a>
             <hr class="navbar-divider">
-            <div class="navbar-item"><strong>Asciidoctor.js</strong></div>
-            <a class="navbar-item" href="https://github.com/asciidoctor/asciidoctor.js" target="_blank" rel="noopener">Source</a>
-            <a class="navbar-item" href="https://github.com/asciidoctor/asciidoctor.js/issues" target="_blank" rel="noopener">Issues</a>
+            <div class="navbar-item"><strong>Extended Converters</strong></div>
+            {{!--
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoctor-pdf::index.adoc')}}">Asciidoctor PDF</a>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoctor-epub3::index.adoc')}}">Asciidoctor EPUB3</a>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoctor-reveal.js::index.adoc')}}">Asciidoctor reveal.js</a>
+            --}}
+            <a class="navbar-item" href="#">Asciidoctor reveal.js</a>
+          </div>
+        </div>
+        <div class="navbar-item has-dropdown is-hoverable">
+          <div class="navbar-link">Tooling</div>
+          <div class="navbar-dropdown">
+            <div class="navbar-item"><strong>Build Plugins</strong></div>
```

Comment:
How about `Build Support` or `Build Integration`? I don't feel "Plugin" adds any meaning here.

Reply:
I'm fine with "Plugins", both Gradle and Maven are using the term "Plugin".

Reply:
What's funny is that I actually had Build Integrations first, then I changed it. I can change it back and we can see if it feels right. The reason I choose "Plugins" is for the reason Guillaume cited (https://maven.apache.org/plugins/index.html and https://plugins.gradle.org/). Let me know if you still want me to change it.

Reply:
The wierd element here for me is Asciidoctlet, but I admit the wording is complex and keeping it simple may be the best. No problem with Plugin.

Reply:
That's a good point. It's a tricky one!